src/utils/constants.js

```diff
@@ -21,8 +21,8 @@ export const HEX_VALUE_REGEX = /(0x)?[0-9A-Fa-f]{40}/
 
 export const COLOR_SCHEME_DEFAULT = [
   '#0be1b1', // green
-  '#e01563', // red
   '#f2cc0a', // yellow
+  '#e01563', // red
```

Comment:
This order was set to have the created outcomes always be:
- Positive Outcome (Green)
- Negative Outcome (Red)

until we can decide the color for the outcomes we should keep it like that. Otherwise Yes/No Outcomes will be green/yellow